### PR TITLE
[DELIVERABLE 1] Development Plan : Add Section 1,2,3 Add Mya Reflection

### PR DESCRIPTION
**Pull request by mmyaaaaa**

### CODE REVIEW

File: docs/DevelopmentPlan/DevelopmentPlan.tex [hunk 1]
```diff
@@ -38,22 +38,16 @@
...
-\wss{State whether there is IP to protect.  If there is, point to the agreement.
-  All students who are working on a project that requires an IP agreement are also
-required to sign the ``Intellectual Property Guide Acknowledgement.''}
+No intellectual property to protect.
 
 \section{Copyright License}
 
-\wss{What copyright license is your team adopting.  Point to the license in your
-repo.}
+The code in this repo is provided free to use under the MIT License.
```
Comment (Sevhena) [inline docs/DevelopmentPlan/DevelopmentPlan.tex:48]: Conflicts with the license @tbrar06 is proposing

**Review by Sevhena — CHANGES_REQUESTED** [review]
Please look at above comment

**Comment by Sevhena** [comment]
@mmyaaaaa Please do the changes mentioned above and rename the pull request with the appropriate naming convention ( [DELIVERABLE #] <Changed Doc>: Add Section <#s> ).

**Review by Ayushi1972 — APPROVED** [review]
Looks good. The license will be updated after @tbrar06 Pull Request gets approved.

**Review by mmyaaaaa — COMMENTED** [review]
Changes done ready to merge

**Review by Sevhena — APPROVED** [review]
LGTM

**Pull request merged**